index.js

```diff
@@ -1113,12 +1113,16 @@
 
   //# equals :: (a, b) -> Boolean
   //.
-  //. Returns `true` if its arguments are of the same type and equal according
-  //. to the type's [`fantasy-land/equals`][] method; `false` otherwise.
+  //. Returns `true` if the two arguments given are structurally equal. When
+  //. comparing two values, equality is determined based on:
   //.
-  //. `fantasy-land/equals` implementations are provided for the following
-  //. built-in types: Null, Undefined, Boolean, Number, Date, RegExp, String,
-  //. Array, Arguments, Error, Object, and Function.
+  //. - The types of the two values. Values of differing types unequal.
+  //. - The [`fantasy-land/equals`][] method of the first value when present.
+  //. - For Null, Undefined, Boolean, Number, Date, RegExp, String, Array,
+  //.   Arguments, Error, Object, and Function, Sanctuary Type Classes provides
+  //.   `fantasy-land/equals` implementations.
+  //. - Any other types are structurally compared by recursively comparing the
+  //.   values at each of the keys.
```

Comment:
I propose the following description:

```javascript
  //. Returns `true` if its arguments are equal; `false` otherwise.
  //.
  //. Specifically:
  //.
  //.   - Arguments of different types are unequal.
  //.
  //.   - If the first argument has a [`fantasy-land/equals`][] method,
  //.     that method is invoked to determine whether the arguments are
  //.     equal (`fantasy-land/equals` implementations are provided for the
  //.     following built-in types: Null, Undefined, Boolean, Number, Date,
  //.     RegExp, String, Array, Arguments, Error, Object, and Function).
  //.
  //.   - Otherwise, the arguments are equal if their
  //.     [entries][`Object.entries`] are equal (according to this algorithm).
```

```javascript
//. [`Object.entries`]:         https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries
```